starter-code/public/index.html

```diff
@@ -79,19 +79,20 @@
     <footer>
       &copy; 2017 Code Fellows | Static Blog | Happy Ipsum!
     </footer>
-    <!-- TODO: Be sure to include the routing functionality with the correct script files.
+    <!-- DONE: Be sure to include the routing functionality with the correct script files.
                Don't forget that the order of your scripts matters - variables have to be
                declared before you can reference them! -->
 
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
     <script src="/vendor/scripts/handlebars.js"></script>
     <script src="/vendor/scripts/marked.js"></script>
     <script src="vendor/scripts/highlight.pack.js"></script>
+    <script src="https://cdn.rawgit.com/visionmedia/page.js/master/page.js"></script>
     <script src="/scripts/models/article.js"></script>
     <script src="/scripts/views/articleView.js"></script>
     <script src="/scripts/controllers/articleController.js"></script>
-
     <script src="/scripts/controllers/aboutController.js"></script>
+    <script src="/scripts/controllers/routes.js"></script>
```

Comment:
Debugging for something as small as a script tag being in the wrong spot is frustrating. But you found it! And next time it will be one of your first checks! (I have done it so many times :)